server/database/schemas/user.schema.js

```diff
@@ -2,18 +2,58 @@ import { Schema } from 'mongoose';
 import sha256 from 'sha256';
 
 const userSchema = new Schema({
-  hashedPassword: { type: String, required: true },
-  email: { type: String, required: true },
-  avatarUrl: { type: String },
-  avatarType: { type: String },
-  firstName: { type: String },
-  lastName: { type: String },
-  dob: { type: Date },
-  bio: { type: String },
-  address: { type: String },
-  phone: { type: String },
-  gender: { type: Number },
-  addedInfo: { type: [Object] }
+  hashedPassword: {
+    type: String,
+    required: true,
+  },
+  email: {
+    type: String,
+    required: true,
+  },
+  avatar_url: {
```

Comment:
Based on a standard guide of ESLint, we should follow camelCase for the naming @DaleCS @ellamaolson 
More info about this rule: **https://eslint.org/docs/rules/camelcase**

```suggestion
  avatarUrl: {
```

Reply:
These are the database fields that will be saved in the database and they don't necessarily need to be variables that we declare in our code. Snake case avoids problems in queries that are case sensitive.